catalogd/testdata/catalogs/test-catalog.Dockerfile

```diff
@@ -1,6 +0,0 @@
-FROM scratch
-COPY test-catalog /configs
-
-# Set DC-specific label for the location of the DC root directory
-# in the image
-LABEL operators.operatorframework.io.index.configs.v1=/configs
```

Comment:
@tmshort for we do not break the bumper: https://github.com/openshift/release/pull/61681

Reply:
I'm actually thinking we may want to keep this Dockerfile around until https://github.com/openshift/release/pull/61681 is merged.

Reply:
Done, I reverted this one